source/common/ssl/context_impl.cc

```diff
@@ -356,9 +325,7 @@ std::string ContextImpl::getSerialNumber(X509* cert) {
   if (char_serial_number != nullptr) {
     std::string serial_number(char_serial_number);
     OPENSSL_free(char_serial_number);
-#ifdef OPENSSL_IS_BORINGSSL
     std::transform(serial_number.begin(), serial_number.end(), serial_number.begin(), ::toupper);
```

Comment:
Note: this is here only for backward-compatibility with existing systems. Does any of your internal systems rely on serial number being uppercase or can we remove this?